### PR TITLE
Form page header

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

**Review by ssilvert — COMMENTED** [review]
@edewit 

Per our conversation yesterday, let's use camel case translation keys.  That way:

- We quickly know if translations are broken or missing.
- We know that all keys theoretically exist in the English translation files, which makes it easier to translate to new languages.

**Comment by edewit** [comment]
changing keys in #88

File: src/components/page-header/FormPageHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+export const FormPageHeader = ({ titleKey, subKey }: FormPageHeaderProps) => {
+  const { t } = useTranslation();
+  const { enabled } = useContext(HelpContext);
+  return (
+    <>
+      <PageSection variant="light">
+        <TextContent>
+          <Text component="h1">{t(titleKey)}</Text>
+          {enabled && t(`help:${subKey}`)}
```
Comment (srambach) [inline src/components/page-header/FormPageHeader.tsx:24]: Let's also wrap this line in a `<Text>` to get the correct vertical spacing.

**Comment by edewit** [comment]
going to become one with #92

**Pull request closed without merging**